gms/gossiper.cc

```diff
@@ -797,6 +797,18 @@ std::set<inet_address> gossiper::get_live_members() {
     return live_members;
 }
 
+size_t gossiper::get_live_members_count() const {
```

Comment:
git grep says that there's one more usage of `get_live_members().size()` idiom in `service/storage_service.cc`, perhaps it should be migrated to `get_live_members_count()` as well.

Reply:
Good catch, not sure why I didn't find it before. Changed.